mlflow/ml-package-versions.yml

```diff
@@ -654,6 +654,7 @@ langchain:
           "numexpr",
           "hf_transfer",
           "langchain-core!=0.1.39", # https://github.com/langchain-ai/langchain/issues/19947
+          "typing_extensions>=4.6.0",
```

Comment:
The docs mention that this was introduced in 4.7.0, which is odd. https://typing-extensions.readthedocs.io/en/latest/index.html#typing_extensions.cast